firmware/network/border_router_node/border_router_node.c

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2022 Mesh4all <mesh4all.org>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * @brief  Border Radio node.
+ *
+ * @author  RocioRojas <rociorojas391@gmail.com>
+ */
+#include <stdio.h>
+#include <string.h>
+#include "border_router_node.h"
+
+int dummy_function (void){
+
+    return 0;
+}
```

Comment:
This is doing nothing

Reply:
Yes, if you read the doc.txt, the important thing is the Makefile.dep